frontend/src/pages/calculatedResults/CalculatedResultsElements.js

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+import 'antd/dist/antd.css';
+import { Tabs } from 'antd';
+
+export const FixedInstallmentsContainer = styled.div`
+  background: linear-gradient(
+    180deg,
+    rgba(173, 229, 247, 0) 4.98%,
+    rgba(147, 228, 239, 0.0618911) 42.46%,
+    rgba(6, 223, 200, 0.4) 74.66%,
+    #5e60ce 98.46%
+  );
+  height: auto;
+  min-height: 90vh;
+  padding-top: 10vh;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
```

Comment:
(optional) For the future projects, when you use some UI lib like AntD, then you should stick to its design system using built-in features to avoid functionality duplication, e.g. https://ant.design/components/grid/

But it would be time consuming to change that now, so I'm leaving that as an optional

Reply:
thanks I'll keep that in mind :)